### PR TITLE
vendor: Weekly update

### PR DESCRIPTION
**Pull request by sameo**
Most importantly we need the ciao qemu huge pages changes:

https://github.com/ciao-project/ciao/pull/1449

The ciao vendoring brings:

11d484b ciao-controller: Make CNCI volume ephemeral
8ff0fc2 sqlite3db: Use sqlite implicit transactions
696afec ssntp: Remove RESTART and STOP from documentation
ab276fc ssntp: Remove STOP and RESTART commands
ea931ac test-cases: Run Example tests
79af066 QEMU: Knobs: Huge Page Support: Add support for huge pages
84e3725 bat: Delete test tenants
615c726 ciao-controller: add support for tenant deletion
4a73900 ciao-cli: add support for tenant delete
2efb224 bat: add BAT tests for tenant commands
8f822c0 ciao-cli: add support for tenant create
939e448 ciao-controller: add api to create new tenants
128e810 ciao-controller: rework AddTenant() to include a tenant config struct
163e901 vendor: Remove gophercloud
ff02a86 ciao-cli: Stop using gophercloud datatypes for volume operations
865ca20 ciao-controller: Stop using pointers to strings in volume API structs
9fa4079 ciao-cli: Stop using gophercloud datatypes for image operations
6b75dca ciao-cli: Remove "volume update"
a21af5f travis: Use if rather than test for conditional stesps
2412081 travis: Enable unused linter
32561ba travis: Lint in parallel to build and test steps
34d2c75 misc: Remove remaining unused code as found by "unused" linter
673ef61 docs: Remove more unneeded documentation
95d38b4 README: Update the top level README.md file
1a0ed7d bat: Moved the documentation into the source
83af79f Docs: Remove out of date or duplicated documentation
5acc01a vfio: Add ability to pass VFIO devices to qemu
0ddd70b ciao-project: Cleanup all remaining references to 01org/ciao
6a4f161 README: Update badges
0801943 singlevm: fix ciao src path
55c5b70 Build: Fix the build after repo move.
153da39 travis: Update .travis.yml for new repo
a673001 ciao-cli: remove stray debug print

Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>

### CODE REVIEW

**Comment by sameo** [comment]
cc @mcastelino

File: Gopkg.toml [hunk 1]
```diff
@@ -129,3 +125,7 @@
 [[constraint]]
   branch = "master"
   name = "golang.org/x/sys"
+
+[[constraint]]
+  name = "github.com/sirupsen/logrus"
+  version = "1.0.3"
```
Comment (grahamwhaley) [inline Gopkg.toml]: I presume the pinning of logrus was deliberate btw.

**Comment by jodh-intel** [comment]
Could you add the shortlog for all the changes (to help debug any potential issues in the future)?

**Comment by sameo** [comment]
@jodh-intel I may be confused but I'm vendoring, not releasing. Which shortlog are you thinking about ?

**Comment by jodh-intel** [comment]
Hi @sameo - I just mean adding the following to the commit message so it's clear what was added to vc from ciao in this re-vendor:

```
cd $GOPATH/src/github.com/01org/ciao
old=9d6632430ecf42a35c23b7c6cdc560d5116d9483
new=cf2e779166b422dc85d28c733ee8442e597bb810
git log --no-merges --abbrev-commit --pretty=oneline "${old}..${new}"
```

**Comment by sboeuf** [comment]
LGTM

**Pull request merged**